kotlin-remix-run-router/src/main/generated/remix/run/router/RevalidationState.kt

```diff
@@ -0,0 +1,11 @@
+@file:JsModule("@remix-run/router")
+@file:JsNonModule
+
+@file:Suppress(
+    "NON_EXTERNAL_DECLARATION_IN_INAPPROPRIATE_FILE",
+)
+
+package remix.run.router
+
+
+typealias RevalidationState = Any /* "idle" | "loading" */
```

Comment:
Conversion for unions isn't implemented?

Reply:
It is marked in TODO list in README for future implementation